pkgs/os-specific/linux/rtl88x2bu/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, kernel, bc }:
+stdenv.mkDerivation rec {
+  name = "rtl88x2bu-git-${kernel.version}-${version}";
+  version = "2020-05-19";
```

since you're not fetching from an official release, current paradigm is prefix with unstable
```suggestion
  version = "unstable-2020-05-19";
```